yarn.lock

```diff
@@ -4155,7 +4155,7 @@ boolbase@^1.0.0, boolbase@~1.0.0:
   resolved "https://registry.yarnpkg.com/boolbase/-/boolbase-1.0.0.tgz#68dff5fbe60c51eb37725ea9e3ed310dcc1e776e"
   integrity sha1-aN/1++YMUes3cl6p4+0xDcwed24=
 
-bootstrap@4.5.3, bootstrap@^4.3.1:
+bootstrap@^4.3.1:
```

Comment:
Should we use ^4.6.2? To ensure they're on latest?

Reply:
Good point. I can do that. That dependency refers to the docs. They use some features directly that weren't being used by the uikit. However, what I did copy into the uikit is 4.5.1